packages/core/scripts/local/getHistoricalPrice.js

```diff
@@ -11,11 +11,12 @@
  *         --network mainnet_mnemonic
  *         --identifier USDBTC
  *         --time 1601503200
+ *         --ancillaryData 0x123abc
  */
 const { fromWei } = web3.utils;
 const { createReferencePriceFeedForFinancialContract, Networker } = require("@uma/financial-templates-lib");
 const winston = require("winston");
-const argv = require("minimist")(process.argv.slice(), { string: ["identifier", "time"] });
+const argv = require("minimist")(process.argv.slice(), { string: ["identifier", "time", "ancillaryData"] });
```

Comment:
@smb2796 FYI this will be a new CLI flag that the user can specify

Reply:
Cool, thanks for the heads up. @Reinis-FRP for visibility.